core/src/main/java/org/elasticsearch/cluster/metadata/MetaData.java

```diff
@@ -719,9 +691,8 @@ public MetaData readFrom(StreamInput in) throws IOException {
         }
         int customSize = in.readVInt();
```

Comment:
We have `readNamedWriteableList` and `writeNamedWriteableList` but it probably isn't worth it to use them here because you don't have lists. Still, it is nice that they standardize writing the size and all.